SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2339,30 +2339,6 @@ \subsection{Alignment}
 \end{minted}
 \normalsize
 
-Consecutive typedefs should be aligned. If white spaces need to be added to a
```

Comment:
Awesome!

Reply:
🥇   This is my primary motivation for working so hard on this!